binheap.py

```diff
@@ -74,7 +74,7 @@ def _heap_down_pop(self):
                     else:
                         return
                 else:
-                    if self._left_child_value > self._storage[index]:
+                    if self._left_child_value(index) > self._storage[index]:
```

Comment:
If this was a bug that was found, write the test that would have failed before the change.